hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -750,6 +757,22 @@ public PropertyBuilder setCommitTimezone(HoodieTimelineTimeZone timelineTimeZone
       return this;
     }
 
+    public PropertyBuilder set(String key, Object value) {
+      if (HoodieTableConfig.PERSISTED_CONFIG_LIST.contains(key)) {
+        this.others.put(key, value);
+      }
+      return this;
+    }
+
+    public PropertyBuilder set(Map<String, Object> props) {
+      for (Map.Entry<String, Object> entry : props.entrySet()) {
```

Comment:
can we go through PERSISTED_CONFIG_LIST here in this for loop and in next line check if its part of incoming props list. iterating through a smaller list is better :)

Reply:
done.